internal/tools/clickhouse_create_table.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE indexer_cache (
```

Comment:
Should we use ORM like https://pkg.go.dev/gorm.io/gorm#section-readme or we skip it for now

Reply:
not sure if the ORM will work with clickhouse